docs/taskbackground.rst

```diff
@@ -35,23 +65,126 @@ The tree of the above event looks something like
+To measure the :math:`t\bar{t}` cross-section, we use an observable that approximately reconstructs the top quark mass. 
+This is done using the following steps:
 
-Machine Learning Component
+#. Filter the events using the criteria explained above (including requiring at least four jets and exactly one lepton)
+#. Calculate all possible combinations of three jets from all jets in each event (utilizes ``ak.combinations``)
+#. Ensure that there is at least one b-tagged jet in each candidate combination (``btagCSVV2 > B_TAG_THRESHOLD``)
+#. Choose the combination with the largest combined transverse momentum (:math:`p_T`)
+#. Calculate combined mass of trijet system
+
+Successful top mass reconstruction can be inferred from the below histogram, which is peaked around the top mass (:math:`\approx 173 GeV`). 
+We can also see that our selection criteria worked, since the majority of events in the histogram are from the :math:`t\bar{t}` sample.
+
+.. image:: images/analysis.png
+  :width: 80%
+  :alt: Top mass distribution demonstrating successful top mass reconstruction.
+
+5. Statistical Model
+---------------------------------------------------------------
+We want to develop a statistical model, parameterized by some physical parameters :math:`\vec{\alpha}`. 
+We have one parameter of interest, the :math:`t\bar{t}` cross-section, and a handful of *nuisance parameters*, which are free physics parameters that are not of interest in this analysis. 
+Changing any nuisance parameter can have an effect on the expected rate of a physics process and/or distribution shapes, leading to systematic uncertainties. 
+In general, we model the effects of changing nuisance parameters by generating Monte Carlo selections for different variations of parameters. 
+Once we have these, we can utilize different interpolation strategies to construct a continuous parameterization. 
+
+Generally, and inclusive of this analysis example, we consider three different variations: one "nominal" parameter setting, where the nominal setting is chosen based on previous experience and studies, one "up" sample, where the parameter is increased by :math:`1\sigma`, and one "down" sample, where the parameter is decreased by :math:`1\sigma`. 
+The standard uncertainties of the parameters often originate from auxiliary measurements, which necessitate the inclusion of additional constraint terms in the statistical model. 
+Using the up/nominal/down measurements, an interpolation can be made (for each bin) so that a model prediction can be provided for any parameter value:
+
+.. image:: images/interpolation.png
+  :width: 60%
+  :alt: Example of interpolation for model predictions, obtained from `CERN-OPEN-2012-016 <https://cds.cern.ch/record/1456844>`_.
+  
+The above example was obtained from `CERN-OPEN-2012-016 <https://cds.cern.ch/record/1456844>`_.
```

Comment:
We should add here explicitly that we are building a statistical model in HistFactory format (which is what this document refers to). The details described above (which are helpful I think) are all implicit in this HistFactory format.